website/config/team.js

```diff
@@ -1,276 +1,282 @@
+        avatarUrl: 'https://avatars.githubusercontent.com/u/4197225?v=4',
       },
       {
-        "name": "Yong Li",
-        "username": "liyong",
-        "githubUsername": "chnliyong",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/4197225?v=4"
+        name: 'Chen Jin',
+        username: 'Jaycean',
+        githubUsername: 'Jaycean',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/36916582?v=4',
       },
       {
-        "name": "Chen Jin",
-        "username": "Jaycean",
-        "githubUsername": "Jaycean",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/36916582?v=4"
+        name: 'ZhengSong Tu',
+        username: 'tzssangglass',
+        githubUsername: 'tzssangglass',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/30819887?v=4',
       },
       {
-        "name": "ZhengSong Tu",
-        "username": "tzssangglass",
-        "githubUsername": "tzssangglass",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/30819887?v=4"
+        name: 'Xingwang Li',
+        username: 'okaybase',
+        githubUsername: 'okaybase',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/75366457?v=4',
       },
       {
-        "name": "Xingwang Li",
-        "username": "okaybase",
-        "githubUsername": "okaybase",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/75366457?v=4"
+        name: 'Ling Samuel',
+        username: 'lingsamuel',
+        githubUsername: 'lingsamuel',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/14567045?v=4',
       },
       {
-        "name": "Ling Samuel",
-        "username": "lingsamuel",
-        "githubUsername": "lingsamuel",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/14567045?v=4"
+        name: 'Yeliang Wang',
+        username: 'wang-yeliang',
+        githubUsername: 'wang-yeliang',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/19886745?v=4',
       },
       {
-        "name": "Yeliang Wang",
-        "username": "wang-yeliang",
-        "githubUsername": "wang-yeliang",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/19886745?v=4"
+        name: 'Shivam Singh',
+        username: 'singhsh',
+        githubUsername: '1502shivam-singh',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/57267960?v=4',
       },
       {
-        "name": "Shivam Singh",
-        "username": "singhsh",
-        "githubUsername": "1502shivam-singh",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/57267960?v=4"
+        name: 'Bisakh Mondal',
+        username: 'bisakh',
+        githubUsername: 'bisakhmondal',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/41498427?v=4',
       },
       {
-        "name": "Bisakh Mondal",
-        "username": "bisakh",
-        "githubUsername": "bisakhmondal",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/41498427?v=4"
+        name: 'Yilin Zeng',
```

Comment:
@Baoyuantop Those formatting changes are made automatically or only your local configuration?

Reply:
Automatically 🤣

Reply:
I haven't had time to push the PR of reformatted code yet.

Reply:
OK

Reply:
We may need a PR to format all the code.